### PR TITLE
Minor updates: Made grammar corrections and update image in make your first request.md 

### PR DESCRIPTION
**Pull request by fakela**

### CODE REVIEW

File: get-started/make-your-first-request.md [hunk 1]
```diff
@@ -38,24 +38,24 @@ To create an API key:
 ## Get Your Sample Code
 ![Mindee Sample code located under the API reference tab on the right](https://files.readme.io/424a9f3-expense-reciept-sample-code.png "sample code")
 
-Sample codes in several languages and command line tool are provided by Mindee, and they are ready to use by simply copying and pasting them into your coding environment, application, or command line tool. To make use of this:
+Sample code in several languages and command-line tools are provided by Mindee, and they are ready to use by simply copying and pasting them into your coding environments, applications, or command-line tools. To make use of this:
```
Comment (fharper) [inline get-started/make-your-first-request.md:41]: I don't think `command-line` takes a dash

File: get-started/make-your-first-request.md [hunk 2]
```diff
@@ -38,24 +38,24 @@ To create an API key:
...
 
 1. On the left side of the screen, click on **Documentation**.
 2. Under the **API Reference** tab, you'll find the **Sample code** section in a number of popular languages or command line tool. Select according to your desired needs.
 3. Select the API key you wish to use.
-4. Copy and paste the code below into your application, coding environment or command line interface. You're ready to go!
+4. Copy and paste the code below into your applications, coding environments or command-line interface. You're ready to go!
```
Comment (fharper) [inline get-started/make-your-first-request.md:46]: Same comment about command-line

File: get-started/make-your-first-request.md [hunk 3]
```diff
@@ -38,24 +38,24 @@ To create an API key:
...
-- Using Use my camera
+- Uploading a **New document**, or
+- Making use of the **Use my camera** feature
 
-### Using New Document
+### Uploading New Document
 With the **New document** feature located at the bottom of the live Interface, you can upload the documents that you want to the Live Interface and this will appear below. To do this, click on the **New document button** and select on the image document you want. You can only upload one document at a time. However, to make uploads faster, you can drag and drop an image document in the image frame.
 
-### Using Use my Camera 
+### Use My Camera
```
Comment (fharper) [inline get-started/make-your-first-request.md:58]: Should this header be the same as the text from the list before?

**Review by fharper — COMMENTED** [review]
I added some questions

File: get-started/make-your-first-request.md (hunk 1 above)
Comment (fakela) [reply get-started/make-your-first-request.md:41]: My editor kept showing me command line with - 
But I can remove that

File: get-started/make-your-first-request.md (hunk 3 above)
Comment (fakela) [reply get-started/make-your-first-request.md:58]: I honestly thought about this as well, I will change the header since we both had the same thought

**Review by fharper — APPROVED** [review]
LGTM

**Pull request merged**